src/linked_lists.c

```diff
@@ -44,6 +44,21 @@ t_lst_elem	*lst_create(void *content, size_t content_size)
 	return (new);
 }
 
+t_lst_elem	*lst_create_no_malloc(void *content)
+{
+	t_lst_elem		*new;
+
+	if (!(new = (t_lst_elem *)malloc(sizeof(t_lst_elem))))
+		return (NULL);
+	if (!content)
```

Comment:
This test is useless cause if content is not null, we put and if content == null, we put null to new->content which mean, content.